src/types/orchestrationTypes.ts

```diff
@@ -65,3 +65,224 @@ export interface CallHttpOptions {
+     * retry options.
+     *
+     * @param name The name of the activity function to call.
+     * @param retryOptions The retry options for the activity function.
+     * @param input The JSON-serializable input to pass to the activity
+     * function.
+     */
+    callActivityWithRetry(name: string, retryOptions: RetryOptions, input?: unknown): Task;
+
+    /**
+     * Calls an operation on an entity, passing an argument, and waits for it
+     * to complete.
+     *
+     * @param entityId The target entity.
+     * @param operationName The name of the operation.
+     * @param operationInput The input for the operation.
+     */
+    callEntity(entityId: EntityId, operationName: string, operationInput?: unknown): Task;
+
+    /**
+     * Send a signal operation to a Durable Entity, passing an argument, without
+     * waiting for a response. A fire-and-forget operation.
+     *
+     * @param entityId ID of the target entity.
+     * @param operationName The name of the operation.
+     * @param operationInput (optional) input for the operation.
+     */
+    signalEntity(entityId: EntityId, operationName: string, operationInput?: unknown): void;
+
+    /**
+     * Schedules an orchestration function named `name` for execution.
+     *
+     * @param name The name of the orchestrator function to call.
+     * @param input The JSON-serializable input to pass to the orchestrator
+     * function.
+     * @param instanceId A unique ID to use for the sub-orchestration instance.
+     * If `instanceId` is not specified, the extension will generate an id in
+     * the format `<calling orchestrator instance ID>:<#>`
+     */
+    callSubOrchestrator(name: string, input?: unknown, instanceId?: string): Task;
+
+    /**
+     * Schedules an orchestrator function named `name` for execution with retry
+     * options.
+     *
+     * @param name The name of the orchestrator function to call.
+     * @param retryOptions The retry options for the orchestrator function.
+     * @param input The JSON-serializable input to pass to the orchestrator
+     * function.
+     * @param instanceId A unique ID to use for the sub-orchestration instance.
+     */
+    callSubOrchestratorWithRetry(
+        name: string,
+        retryOptions: RetryOptions,
+        input?: unknown,
+        instanceId?: string
+    ): Task;
+
+    /**
+     * Schedules a durable HTTP call to the specified endpoint.
+     *
+     * @param options The HTTP options object
+     */
+    callHttp(options: CallHttpOptions): Task;
+
+    /**
+     * Restarts the orchestration by clearing its history.
+     *
+     * @param input The JSON-serializable data to re-initialize the instance with.
+     */
+    continueAsNew(input: unknown): void;
+
+    /**
+     * Creates a durable timer that expires at a specified time.
+     *
+     * All durable timers created using this method must either expire or be
+     * cancelled using `TimerTask.cancel()` before the orchestrator
```

Comment:
I appreciate that you cleaned up the docs in several places, but in the future please avoid _any_ change that you possibly can when moving code. This is to speed up the review of these PRs and ensure things don't slip in on accident

Reply:
Sounds good, thank you for the feedback.